packages/babel-helpers/src/helpers.js

```diff
@@ -620,6 +620,10 @@ helpers.interopRequireWildcard = helper("7.0.0-beta.0")`
       return obj;
     }
 
+    if (obj === null || (typeof obj !== "object" && typeof obj !== "function")) {
```

Comment:
What about adding tests for this? It would be easier to understand how this function should work, and prevent possible issues in the future.

Reply:
btw for test case, you may refer to `packages/babel-plugin-proposal-dynamic-import/test/fixtures/commonjs/*`. You will find an example for function module `exec-interop` and newly added `exec-interop-string`, `exec-interop-null`